docs/pipeline-api.md

```diff
@@ -12193,7 +12193,7 @@ string
 </tr>
 <tr>
 <td>
-<code>taskServiceAccountName</code><br/>
+<code>serviceAccountName</code><br/>
```

Comment:
This document is generated, it cannot be edited manually - this change is what's causing the CI failutr